.pax/post-packaging.sh

```diff
@@ -118,28 +156,55 @@ if [ "${UC_CURR_PWD}" != "${CURR_PWD}" ]; then
   rm -fr "${UC_CURR_PWD}"
 fi
 
-# get the final build result
-ZOWE_SMPE_PAX="AZWE${FMID_VERSION}/gimzip/AZWE${FMID_VERSION}.pax.Z"
-if [ ! -f "${CURR_PWD}/zowe/${ZOWE_SMPE_PAX}" ]; then
-  echo "[$SCRIPT_NAME][ERROR] cannot find build result ${ZOWE_SMPE_PAX}"
+# save current build log directory, will be placed in artifactory
+cd "${SMPE_BUILD_LOG_DIR}"
+pax -w -f "${CURR_PWD}/smpe-build-logs.pax.Z" *
+
+# find the final build results
+cd "${SMPE_BUILD_SHIP_DIR}"
+SMPE_FMID_ZIP="${FMID}.zip" # keep in sync with smpe/bld/smpe-pd.sh
+if [ ! -f "${SMPE_FMID_ZIP}" ]; then
+  echo "[$SCRIPT_NAME][ERROR] cannot find SMPE_FMID_ZIP build result '${SMPE_FMID_ZIP}'"
   exit 1
 fi
-ZOWE_SMPE_README="AZWE${FMID_VERSION}/gimzip/AZWE${FMID_VERSION}.readme.txt"
-if [ ! -f "${CURR_PWD}/zowe/${ZOWE_SMPE_README}" ]; then
-  echo "[$SCRIPT_NAME][ERROR] cannot find build result ${ZOWE_SMPE_README}"
+SMPE_PTF_ZIP="$(ls -1 ${FMID}.*.zip || true)" # keep in sync with smpe/bld/smpe-service.sh
+if [ ! -f "${SMPE_PTF_ZIP}" ]; then
+  echo "[$SCRIPT_NAME][ERROR] cannot find SMPE_PTF_ZIP build result '${SMPE_PTF_ZIP}'"
+  exit 1
+fi
+SMPE_PD_HTM="${FMID}.htm" # keep in sync with smpe/bld/smpe-pd.sh
+if [ ! -f "${SMPE_PD_HTM}" ]; then
+  echo "[$SCRIPT_NAME][ERROR] cannot find SMPE_PD_HTM build result '${SMPE_PD_HTM}'"
   exit 1
 fi
 
-cd "${CURR_PWD}"
-mv "zowe/${ZOWE_SMPE_PAX}" "zowe-smpe.pax"
-mv "zowe/${ZOWE_SMPE_README}" "readme.txt"
+# if ptf-bucket.txt exists then publish PTF, otherwise publish FMID
+cd "${SMPE_BUILD_SHIP_DIR}"
+if [ -f ${CURR_PWD}/smpe/service/ptf-bucket.txt ]; then
+  tar -cf ${CURR_PWD}/zowe-smpe.tar ${SMPE_PTF_ZIP}
+  # do not alter existing PD in docs, wipe content of the new one
+  rm "${SMPE_BUILD_SHIP_DIR}/${SMPE_PD_HTM}"
+  touch "${SMPE_BUILD_SHIP_DIR}/${SMPE_PD_HTM}"
+else
+  tar -cf ${CURR_PWD}/zowe-smpe.tar ${SMPE_FMID_ZIP}
+  # doc build pipeline must pick up PD for inclusion
+fi
 
-# check what's in logs
-cd "zowe/AZWE${FMID_VERSION}/logs"
-pax -w -f "${CURR_PWD}/smpe-build-logs.pax.Z" *
+# stage build output for upload to artifactory
+cd "${CURR_PWD}"
+mv "${SMPE_BUILD_SHIP_DIR}/${SMPE_FMID_ZIP}"  fmid.zip
+mv "${SMPE_BUILD_SHIP_DIR}/${SMPE_PTF_ZIP}"  ptf.zip
+mv "${SMPE_BUILD_SHIP_DIR}/${SMPE_PD_HTM}" pd.htm
 
 # prepare rename to original name
-cd "${CURR_PWD}"
-echo "mv zowe-smpe.pax AZWE${FMID_VERSION}.pax.Z" > "rename-back.sh.1047"
-echo "mv readme.txt AZWE${FMID_VERSION}.readme.txt" >> "rename-back.sh.1047"
+# leave fixed name for PD to simplify automated processing by doc build
+echo "mv fmid.zip ${SMPE_FMID_ZIP}" > rename-back.sh.1047
+echo "mv ptf.zip ${SMPE_PTF_ZIP}" > rename-back.sh.1047
```

Comment:
On line 202, do you mean overwrite `rename-back.sh.1047`?

In build https://wash.zowe.org:8443/job/zowe-install-packaging/view/change-requests/job/PR-930/24/console:

```
+ cat rename-back.sh
mv ptf.zip AZWE001.TMP0001.zip
+ ./rename-back.sh
```

Only has ptf.zip.

Also you renamed `pd.htm`, do we need to rename it back?

Reply:
Good catch on the overwrite of rename-back.
For the PD, I thought it might be easier for the doc pipeline if we use a fixed file name for PD (if we ever set it up that the doc pipeline pulls the html as input source).
That said, it might be easier on the autoamted test cases as well if we keep fixed names for the FMID & PTF zips. The content of the zips will still have to actual names. 
@John-A-Davies 
What do you think, fixed names in artifcatory (fmid.zip & ptf.zip), or sysmod-specific names (azwe001.zip & azwe001.tmp0001.zip). The filese withing the zip will always be sysmod specific.